PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs

```diff
@@ -86,5 +85,24 @@ private static void GenerateQuestionsFile(string filePath, int numberOfQuestions
                 File.AppendAllLines(filePath, lines);
             }
         }
+        [TestMethod]
+        public void RandomOrderedQuestionLoader()  
+        {
+            string filePath = Path.GetRandomFileName();
+            try
+            {
+                //arrange
+                GenerateQuestionsFile(filePath, 2);
+                //act
+                Question[] firstQuestion = Program.LoadQuestions(filePath);
+                Question[] secondQuestion = Program.LoadQuestions(filePath);
+                bool arrayisequal = firstQuestion.SequenceEqual(secondQuestion);
+                //asset
+                Assert.IsFalse(arrayisequal);
+            }
+            finally
+                File.Delete(filePath);
+
```

Comment:
```suggestion
            finally
            {
                File.Delete(filePath);
            }

```

This is causing errors in your code.